meta/conf/machine/salvator-x-m3.conf

```diff
@@ -0,0 +1,3 @@
+require xt-distro-machine.inc
+
+MACHINEOVERRIDES =. "rcar:rcar-gen3:"
```

Comment:
Your commit message states that this change allows us to distinguish between, say, M3 and H3. I can't see how it is done by this change.